static/scss/shared-styles.scss

```diff
@@ -0,0 +1,13 @@
+img.rounded-profile-image {
+  &.small {
+    height: 43px;
+    border-radius: 50px;
+  }
+
+  &.medium {
+    height: auto !important;
```

Comment:
this was copied from the original style. not sure i want to find out why it's there as a part of this PR